docs/executing/monitoring.rst

```diff
@@ -0,0 +1,16 @@
+.. _monitoring:
+
+========================
+Monitoring
+========================
+
```

Comment:
Thank you, this is a great start! Can you write it a bit more in the style of "snakemake will send the following requests to the wms monitor: ..." and then a table, which lists, and explains each request and the data that is sended along with it.

Reply:
Sorry for requesting an additional change, but I feel that I don't know enough about the api to modify it myself.

Reply:
No problem, I understand that. We can also make changes on panoptes side if you think that this will make things cleaner. For example, I think that the long term plan would be to add full [WES](https://github.com/ga4gh/workflow-execution-service-schemas) support.
- Now we could support only GET requests and show them, but we cannot make POST from panoptes to trigger a snakemake pipeline.
- Another thing that we could refactor before merging is how to keep a unique id for pipelines that we restart. Let's say that a pipeline crashed for whatever reason. When we restart it from the command line it gets a new uuid from panoptes. Is there a unique id generated on snakemake that we could use?

I will make soon an extended table of the current documentation.

Reply:
@johanneskoester Did you have something like this in your mind? Feel free to request new changes.